<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Widget</class>
 <widget class="QWidget" name="Widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Widget</string>
  </property>
  <widget class="QCommandLinkButton" name="commandLinkButton_Folder">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>210</y>
     <width>187</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>打开文件夹</string>
   </property>
  </widget>
  <widget class="QCommandLinkButton" name="commandLinkButton_Web">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>250</y>
     <width>187</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>打开教程页面</string>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>10</y>
     <width>331</width>
     <height>121</height>
    </rect>
   </property>
   <property name="title">
    <string>兴趣爱好</string>
   </property>
   <widget class="QCheckBox" name="checkBox_Read">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>30</y>
      <width>91</width>
      <height>19</height>
     </rect>
    </property>
    <property name="text">
     <string>读书</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="checkBox_Swim">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>60</y>
      <width>91</width>
      <height>19</height>
     </rect>
    </property>
    <property name="text">
     <string>游泳</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="checkBox_Climb">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>30</y>
      <width>91</width>
      <height>19</height>
     </rect>
    </property>
    <property name="text">
     <string>爬山</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="checkBox_Shop">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>90</y>
      <width>91</width>
      <height>19</height>
     </rect>
    </property>
    <property name="text">
     <string>逛街</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="checkBox_Daze">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>60</y>
      <width>91</width>
      <height>19</height>
     </rect>
    </property>
    <property name="text">
     <string>发呆</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="checkBox_Game">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>90</y>
      <width>91</width>
      <height>19</height>
     </rect>
    </property>
    <property name="text">
     <string>游戏</string>
    </property>
   </widget>
  </widget>
  <widget class="QCheckBox" name="checkBox_Tristate">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>160</y>
     <width>91</width>
     <height>19</height>
    </rect>
   </property>
   <property name="text">
    <string>三态显示</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_Show">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>160</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>弹窗显示</string>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
